src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -87,7 +87,7 @@ def _value_export_json(
     # way for the output to be NaN is if the input is invalid or if the sampling
     # function is buggy. Either way, that would be a bug and we can report it by
     # having json throw an error.
-    json.dump(json_out, path.open("w"), allow_nan=False)
+    json.dump(json_out, path.open("w"), allow_nan=False, indent=2)
```

Comment:
`grep -r "indent="` reveals that `indent=4` is the common indent level for ert output.

Reply:
But not for the old `parameters.json`, which uses `indent=0`, so have reverted to that.